File: 0389_Find the Difference/FindtheDifference.java
class Solution {
    public char findTheDifference(String s, String t) {
        char v = 0;
        for(char c1 : s.toCharArray()) v ^= c1;
        for(char c2 : t.toCharArray()) v ^= c2;
        return v;
    }
}

File: 0389_Find the Difference/FindtheDifference.cs
class Solution {
    public char FindTheDifference (string s, string t) {
        if (s == null && t != null && t.Length == 1) {
            return t[0];
        }

        int a = 0;
        int b = 0;
        foreach (var c in s) {
            a ^= (int) c;
        }

        foreach (var c in t) {
            b ^= (int) c;
        }

        a ^= b;

        return (char) a;
    }
}